lib/utils/helpers.coffee

```diff
@@ -47,17 +47,14 @@ exports.stateToString = (state) ->
 		else
 			throw new Error("Unsupported operation: #{state.operation.type}")
 
-exports.sudo = (command, message) ->
-	command = _.union(_.take(process.argv, 2), command)
-	console.log(message)
-
+exports.sudo = (command) ->
 	if os.platform() isnt 'win32'
-		console.log('Type your computer password to continue')
+		console.log('If asked please type your computer password to continue')
 
+	command = _.union(_.take(process.argv, 2), command)
```

Comment:
I have no idea how this used to work, really. 
It was sudo-ing the command that tries to fetch some data using the SDK. 
The change that I've done only sudo-es the command that writes to the drive given all the required info.